legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/model/DataSpaceTemplateExecutableInfo.java

```diff
@@ -0,0 +1,21 @@
+// Copyright 2024 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.generation.analytics.model;
+
+public class DataSpaceTemplateExecutableInfo extends DataSpaceExecutableInfo
+{
```

Comment:
Also seems like this would probably increase the time of computing the anaylsis result. we already compute this i believe for the execution contexts keys so i dont think we should do this twice but just have a pointer.

Reply:
1. It would be consistent with other DataSpaceExecutableInfo s
2. If this won't be done in the engine, it would have to be done Studio